### ISSUE TITLE
Adding a message text below multistep form

### ISSUE THREAD

**Issue by paaljoachim** [munirkamal/gutenberg-forms#83]
I am now testing out the multistep form.
I want to add some text below the form saying something about the multistep form.

![Screen Shot 2020-03-28 at 15 52 31](https://user-images.githubusercontent.com/5323259/77825852-43d80c00-710c-11ea-9f8f-b70392304bdb.png)

I was wondering which field to add then I remembered that I could add the regular Heading or paragraph Gutenberg block to add my own text. 

As I went hunting for a heading and paragraph block I saw this:
![Screen Shot 2020-03-28 at 15 54 21](https://user-images.githubusercontent.com/5323259/77825895-7da91280-710c-11ea-901f-f56c72b02c0b.png)

![Screen Shot 2020-03-28 at 15 55 17](https://user-images.githubusercontent.com/5323259/77825904-8ac60180-710c-11ea-89b0-dde46646a545.png)

![Screen Shot 2020-03-28 at 15 54 34](https://user-images.githubusercontent.com/5323259/77825911-8c8fc500-710c-11ea-81c3-2e3446884491.png)

Basically the Gutenberg form blocks repeated over and over again. No other Gutenberg blocks are available.

Again I just want to add some text that are seen in all multistep pages.

**Comment by munirkamal** [comment]
@paaljoachim 

I've just recorded a quick rough screencast to explain to you how it works. Sorry about bad English or background noises.
https://www.loom.com/share/b971ec0caa0b4df1920e0dd2bfb67b93

I hope this makes it clear to you? or I miss understood your issue?

**Comment by paaljoachim** [comment]
Yes that made it clear! Thanks Munir.
I am working on a feedback thread - walk through of the multistep block.
